k9mail/src/main/java/com/fsck/k9/view/MessageHeader.java

```diff
@@ -295,6 +298,13 @@ public void populate(final Message message, final Account account) {
             mContactBadge.setVisibility(View.GONE);
         }
 
+        if (shouldShowSender(message)) {
+            mSenderView.setVisibility(VISIBLE);
```

Comment:
It's already invisible in the xml file. Should I make it visible there and invisible here ?